web/web/notifications/static/notifications.js

```diff
@@ -52,11 +58,26 @@ class Notifications extends React.Component {
         this.setState({inputValueReferences: refs});
     }
 
-    handleNotificationChange = (e, id, email, notificationType) => {
+    handleNotificationChange = (e, id, notificationId, notExist, email, alertTypeId) => {
         let refs = this.state.checkboxReferences;
         refs[id] = e.currentTarget.checked;
         this.setState({checkboxReferences: refs});
-        //this.props.dispatch(updateNotificationPreference(email, notificationType, event.currentTarget.checked));
+        notExist
+        ?
+        this.props.dispatch(action.postNotifications({
+            "alert_type_id" : alertTypeId,
+            "email" : email,
+            "is_active" : refs[id],
+            "created_by" : 1
```

Comment:
Not sure if this can be submitted an issue to address later on, but if you do have time to change this I think "created_by" id should correlate with the current logged in user. I believe their id is stored in userData once they log in, you can also get the id with the token on the flask side.

Reply:
fixed